cst338.xhtml

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
+"http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<link rel="icon" href="imgs/Glogo2_nobg.png" type="image/png" />
+	<!-- External Link to our CSS file -->
+	<link rel="stylesheet" href="styles.css" />
+	<title>CST3338</title>
+	<script src="scripts.js" defer="defer"></script>
+</head>
+<body>
+	
+	<div id="header"></div>
+		<main>
+			<h3>CST338: Software Design</h3>
```

Comment:
Instead of using generic `<h3>` tags, you should just make these into there own `<p>` or `<span>` elements and define your own style via class/id. Then we can alter them better.